diff --git a/node_modules/uniffi-bindgen-react-native/crates/ubrn_cli/src/codegen/templates/CMakeLists.txt b/node_modules/uniffi-bindgen-react-native/crates/ubrn_cli/src/codegen/templates/CMakeLists.txt
index 3a8d733..b2e0a1d 100644
--- a/node_modules/uniffi-bindgen-react-native/crates/ubrn_cli/src/codegen/templates/CMakeLists.txt
+++ b/node_modules/uniffi-bindgen-react-native/crates/ubrn_cli/src/codegen/templates/CMakeLists.txt
@@ -17,10 +17,7 @@ execute_process(
     OUTPUT_VARIABLE UNIFFI_BINDGEN_PATH
     OUTPUT_STRIP_TRAILING_WHITESPACE
 )
-string(REGEX
-  REPLACE "/package\\.json$" ""
-  UNIFFI_BINDGEN_PATH ${UNIFFI_BINDGEN_PATH}
-)
+get_filename_component(UNIFFI_BINDGEN_PATH "${UNIFFI_BINDGEN_PATH}" DIRECTORY)
 
 # Specifies a path to native header files.
 include_directories(
